packages/vector/client.py

```
@@ -0,0 +1,333 @@
+        Returns:
+            bool: True if server is healthy, False otherwise.
+
+        Example:
+            >>> client = QdrantVectorClient(url="http://localhost:6333", collection_name="docs")
+            >>> if client.health_check():
+            ...     print("Qdrant is healthy")
+        """
+        try:
+            self.client.get_collections()
+            logger.debug("Qdrant health check passed")
+            return True
```

Ruff (0.14.1)

Consider moving this statement to an `else` block

(TRY300)